R/aggregate.R

```diff
@@ -0,0 +1,166 @@
+##' This function calculates the robust summarisation for each feature
+##' (protein). Note that the function assumes that the intensities in
+##' input `e` are already log-transformed.
+##'
+##' @title Return the Robust Expression Summary of a matrix
```

Comment:
I would prefer to have the title in the first line.

Reply:
Done